test/parallel/test-url-parse-format.js

```diff
@@ -1059,3 +1059,6 @@ for (const u in parseTests) {
 
   assert.deepStrictEqual(parsed, expected);
 }
+
+// Validate the output of hostname with commas.
+assert.strictEqual(url.parse('x://0.0,1.1/').host, '0.0,1.1');
```

Comment:
Why not adding 

```
'x://0.0,1.1/' :  {
  protocol: 'x:',
  slashes: true,
  auth: null,
  host: '0.0,1.1',
  port: null,
  hostname: '0.0,1.1',
  hash: null,
  search: null,
  query: null,
  pathname: '/',
  path: '/',
  href: 'x://0.0,1.1/'
}
```

to `parseTests` instead?

Reply:
Makes sense. Updated and force-pushed. Thanks!